src/PostgREST/Auth.hs

```diff
@@ -49,10 +50,19 @@ claimsToSQL = map setVar . toList
   returns a map of JWT claims
   In case there is any problem decoding the JWT it returns Nothing.
 -}
-jwtClaims :: Text -> Text -> Maybe JWT.ClaimsMap
-jwtClaims secret input = JWT.unregisteredClaims . JWT.claims <$> decoded
+jwtClaims :: Text -> Text -> NominalDiffTime -> Maybe JWT.ClaimsMap
+jwtClaims secret input time =
+  case claim JWT.exp of
+    Just (Just expires) ->
```

Comment:
Ooh, thank you so much for that `join` method. I spent a lot of time looking for that.